model/src/main/java/org/alfresco/transform/common/TransformerDebug.java

```diff
@@ -180,6 +193,14 @@ String getOptionAndValue(String reference, String key, String value)
                 "  " + key + "=\"" + value.replaceAll("\"", "\\\"") + "\"";
     }
 
+    public void logFragment(String reference, int index, long size)
+    {
+        if (logger.isDebugEnabled())
+        {
+            logger.debug(getPaddedReference(reference) + "  fragment["+index+"] "+fileSize(size));
```

Comment:
I know we don't use SonarLint in this project, but if we intend to fork this for HXPR we may end up using it in the fork.
[Printf-style format strings should be used correctly (java:S3457)
CODE_SMELL  Code Smell  MAJOR  Major](https://rules.sonarsource.com/java/RSPEC-3457)